src/static_canvas.class.ts

```diff
@@ -747,12 +747,17 @@ import { pick } from './util/misc/pick';
      * @return {fabric.Canvas} instance
      * @chainable
      */
-    renderCanvas: function(ctx, objects) {
+    renderCanvas: function (ctx, objects) {
+
+      if (this.destroyed) {
+        return;
+      }
+
       var v = this.viewportTransform, path = this.clipPath;
-      this.cancelRequestedRender();
```

Comment:
this can't go away, if something renders it needs to kill the queued render. it can be moved in `renderAll` it will still keep the functionality intact but needs to be specified for both static_canvas and canvas.

Reply:
> fix(): canvas#toCanvasElement used to kill requested renders https://github.com/fabricjs/fabric.js/commit/b12491dbbb725b00943629bb3e7d4917188cdb29

It fixes a bug

So you mean the case in which there is a requested render and the dev call `renderAll` right?

Reply:
DONE 0ea2761d22bbc7668f042f44ba33c28c2c16a3b5